<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinMain</class>
 <widget class="QMainWindow" name="WinMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>S</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="test_selector">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Throughput</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Responsivness</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_scheduler">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Schedulers for test:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="text">
     <string>Logo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_test">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Test:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_light">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string> Light</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_heavy">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string> Heavy</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_custom">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_custom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="box_1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>86</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>BFQ</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="box_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>86</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CFQ</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="box_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>86</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>NOOP</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="box_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>DEADLINE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clean">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAN</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_test_description">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Descrizione test</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_history">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionInfo_benchmark"/>
    <addaction name="actionCredits"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionInfo_benchmark">
   <property name="text">
    <string>Info test</string>
   </property>
  </action>
  <action name="actionCredits">
   <property name="text">
    <string>Credits</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
